stdlib/task/src/main/ballerina/src/task/scheduler.bal

```diff
@@ -76,7 +76,7 @@ public type Scheduler object {
 
     # Pauses the task.
     #
-    # + return - Returns `task:SchedulerError` if an error is occurred while resuming, nil Otherwise.
+    # + return - `task:SchedulerError` if an error is occurred while pausing, nil otherwise
```

Comment:
```suggestion
    # + return - `task:SchedulerError` if an error is occurred while pausing, nil otherwise
```
Same as above. Also, please refer the template for the correct format of the return statement description.